### ISSUE TITLE
How much data required and how to train on custom speaker data

### ISSUE THREAD

**Issue by shivamani-ans** [NVIDIA/tacotron2#167]
Hi,

I have downloaded LJS speaker pre-trained model and configured it as warm-start for custom speaker data training.

And i have both wave glow models waveglow_old.pt and waveglow_256channels.pt for prediction but prediction is too bad audio is unclear.

Any suggestions on how to proceed further ? do i need to train waveglow model also by considering above waveglow models as warm-start.

**Comment by rafaelvalle** [comment]
Hard to say without seeing the loss and alignments of your tacotron2 trained on your data.

**Comment by shivamani-ans** [comment]
Hi,
Loss = 0.503 (approx) at 200 epoch (4000 steps). I didn't understand about alignments where can i check that ?

**Comment by rafaelvalle** [comment]
You'll need to train longer and anneal the learning rate to reduce the validation loss.

**Comment by shivamani-ans** [comment]
Hi,
I have reduce learning rate from 10e-4 to both 10e-6 and 10e-5 but there are no differences in prediction performance.

1. we took pretrained model (tacotron2_statedict.pt) and our data (1000 samples = 1.5 hr)-> didn't worked at all.
2. we have trained from scratch with our data (1000 samples = 1.5 hr) -> didn't worked at all.

In above both cases while testing prediction on test files we have used both waveglow_256channels and waveglow_old.pt but no differences in output performance in prediction.

And we also trained waveglow model both from scratch and fine-tune using pre-trained but didn't worked at all.

Could you please help us ? what mistake we are committing on our side ?

**Comment by tugstugi** [comment]
@shivamani-ans check whether your WAV files are int or float... If float, training will not work.

**Comment by shivamani-ans** [comment]
Hi,

when we open .wav file in audacity it shows Mono, 16KHz 32-bit float.

To which int format do i need to convert (16-bit PCM or 24-bit PCM) ?

**Comment by Yeongtae** [comment]
Could you share your attention graph?

**Comment by shivamani-ans** [comment]
@rafaelvalle  
could you please re-open this issue, after "anneal the learning rate to reduce the validation loss", there is no difference in prediction output (its complete empty noise).

**Comment by rafaelvalle** [comment]
As I said, it's hard to help without seeing your tensorboard log with loss curves and attention graphs, etc...

**Comment by ST2-EV** [comment]
Hey you can use this [GUI](https://github.com/danklabs/tts_dataset_maker)
